phoenix-core/src/main/java/org/apache/phoenix/schema/stats/StatsCollectionDisabledOnServerException.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.phoenix.schema.stats;
+
+import org.apache.phoenix.exception.SQLExceptionCode;
+import org.apache.phoenix.exception.SQLExceptionInfo;
+
+import java.sql.SQLException;
+
+public class StatsCollectionDisabledOnServerException extends SQLException {
+
+    private static final long serialVersionUID = 1L;
+    private static SQLExceptionCode code = SQLExceptionCode.STATS_COLLECTION_DISABLED_ON_SERVER;
+
+    public StatsCollectionDisabledOnServerException() {
```

Comment:
It's better to pass table name and call setTableNae in SQLExceptionInfo.Builder.

Reply:
The exception is wrapped around by `DoNotRetryIOException` from HBase which prints the region name automatically (which contains table name), hence I didn't add it here.